openedx/core/djangoapps/agreements/models.py

```diff
@@ -22,3 +22,15 @@ class IntegritySignature(TimeStampedModel):
     class Meta:
         app_label = 'agreements'
         unique_together = ('user', 'course_key')
+
+
+class LTIPIISignature(models.Model):
+    """
+    This model stores a user's acknowledgement to share PII via LTI tools in a particular course.
+    """
+    course_key = CourseKeyField(max_length=255, db_index=True)
+    user_id = models.IntegerField()
```

Comment:
I think this is a result of the way I wrote the ticket, but this relationship would be better represented as a ForeignKeyField, like the `user` field of the `IntegritySignature` model above. I know I wrote `user_id` in the ticket, so that might have been confusing.

The reason I wrote `user_id` in the Jira ticket is that, under the hood, Django represents foreign key column names in the database using the format `<field_name>_id`, so a field of `user`, like in the `IntegritySignature` module, would be represented by the `user_id` column. You can take a look at the [Database Representation](https://docs.djangoproject.com/en/4.2/ref/models/fields/#database-representation) documentation for more information.

Sorry about the confusion!